[BUG] SaveActiveUsersInteractor compilation with latest Rocket.Chat.Kotlin.SDK

## Description

SaveActiveUsersInteractor is not compiling anymore due a breaking change in chat.rocket.common.model.User @ Rocket.Chat.Kotlin.SDK due commit 

https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK/commit/86ce9ca441f241c6e09798ce05b6817d2c258f5b#diff-d5048ca527be07710179ff421e146625

## Devices and Versions

Rocket.Chat.Android develop branch

## Steps to reproduce

Pull Rocket.Chat.Android and Rocket.Chat.Kotlin.SDK `develop` branches and `./gradlew clean build` 

## Logs

```
alex@alex:~/dev/Rocket.Chat.Android$ ./gradlew clean build

> Configure project :app 
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
Could not find google-services.json while looking in [src/nullnull/debug, src/debug/nullnull, src/nullnull, src/debug, src/nullnullDebug]
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
Could not find google-services.json while looking in [src/nullnull/release, src/release/nullnull, src/nullnull, src/release, src/nullnullRelease]
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)

> Task :app:compileSdk 
CURRENT DIR: /home/alex/dev/Rocket.Chat.Android/app
SDK DIR: /home/alex/dev/Rocket.Chat.Kotlin.SDK
CURRENT SHA: 47b5981
not dirty
same hash as before 47b5981 = 47b5981
NO BUILD NEEDED

> Task :app:processDebugGoogleServices 
Parsing json file: /home/alex/dev/Rocket.Chat.Android/app/google-services.json

> Task :app:processReleaseGoogleServices 
Parsing json file: /home/alex/dev/Rocket.Chat.Android/app/google-services.json

w: /home/alex/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre7/1.2.41/9e7a6f582de73d9cdc6c56ef4e23604a0ee55768/kotlin-stdlib-jre7-1.2.41.jar: kotlin-stdlib-jre7 is deprecated. Please use kotlin-stdlib-jdk7 instead
w: /home/alex/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre8/1.2.41/7b389de81710eb609752953f38ad6e7de2927292/kotlin-stdlib-jre8-1.2.41.jar: kotlin-stdlib-jre8 is deprecated. Please use kotlin-stdlib-jdk8 instead
w: /home/alex/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre7/1.2.41/9e7a6f582de73d9cdc6c56ef4e23604a0ee55768/kotlin-stdlib-jre7-1.2.41.jar: kotlin-stdlib-jre7 is deprecated. Please use kotlin-stdlib-jdk7 instead
w: /home/alex/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre8/1.2.41/7b389de81710eb609752953f38ad6e7de2927292/kotlin-stdlib-jre8-1.2.41.jar: kotlin-stdlib-jre8 is deprecated. Please use kotlin-stdlib-jdk8 instead
w: Note: Version 5.1.0 of Realm is now available: https://static.realm.io/downloads/java/latest
w: /home/alex/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre7/1.2.41/9e7a6f582de73d9cdc6c56ef4e23604a0ee55768/kotlin-stdlib-jre7-1.2.41.jar: kotlin-stdlib-jre7 is deprecated. Please use kotlin-stdlib-jdk7 instead
w: /home/alex/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre8/1.2.41/7b389de81710eb609752953f38ad6e7de2927292/kotlin-stdlib-jre8-1.2.41.jar: kotlin-stdlib-jre8 is deprecated. Please use kotlin-stdlib-jdk8 instead
w: Note: Processing module RocketChatLibraryModule
w: Note: Processing module RocketChatServerModule
w: Note: Processing class RealmPreferences
w: Note: Processing class RealmUser
w: Note: Processing class RealmSession
w: Note: Processing class RealmSettings
w: Note: Processing class RealmEmail
w: Note: Processing class RealmBasedServerInfo
w: Note: Processing class RealmPublicSetting
w: warning: Supported source version 'RELEASE_7' from annotation processor 'org.jetbrains.kotlin.kapt3.ProcessorWrapper' less than -source '1.8'
w: /home/alex/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre7/1.2.41/9e7a6f582de73d9cdc6c56ef4e23604a0ee55768/kotlin-stdlib-jre7-1.2.41.jar: kotlin-stdlib-jre7 is deprecated. Please use kotlin-stdlib-jdk7 instead
w: /home/alex/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre8/1.2.41/7b389de81710eb609752953f38ad6e7de2927292/kotlin-stdlib-jre8-1.2.41.jar: kotlin-stdlib-jre8 is deprecated. Please use kotlin-stdlib-jdk8 instead
e: /home/alex/dev/Rocket.Chat.Android/app/src/main/java/chat/rocket/android/server/domain/SaveActiveUsersInteractor.kt: (33, 13): No value passed for parameter 'roles'

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 24s
86 actionable tasks: 82 executed, 4 up-to-date
```

--- a/app/src/main/java/chat/rocket/android/server/domain/SaveActiveUsersInteractor.kt
+++ b/app/src/main/java/chat/rocket/android/server/domain/SaveActiveUsersInteractor.kt
@@ -29,7 +29,8 @@ class SaveActiveUsersInteractor @Inject constructor(
                 username = user.username ?: it.username,
                 status = user.status ?: it.status,
                 emails = user.emails ?: it.emails,
-                utcOffset = user.utcOffset ?: it.utcOffset
+                utcOffset = user.utcOffset ?: it.utcOffset,
+                roles = user.roles ?: it.roles
             )
 
             val activeUserList: MutableList<User> =

